virtool/labels/checks.py

```diff
@@ -0,0 +1,14 @@
+import aiohttp.web
+import re
+
+
+async def check_hex_color(req: aiohttp.web.Request):
+    """
+    Check if the Hex color in the request is valid.
+
+    """
+    regex = "^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$"
+
+    match = re.match(regex, req["data"]["color"])
```

Comment:
For `re` it is better to compile the regex  at scope level so it isn't recompiled every time `check_hex_color` is called.

I usually name them with the form `RE_HEX_COLOR`. [See relevant line in another module](https://github.com/virtool/virtool/blob/7f7c7229bccccb26c683ec5b5c432d42668a4d4c/virtool/samples/migrate.py#L22)

https://docs.python.org/3.7/library/re.html#re.compile